src/App.css

```diff
@@ -1,3 +1,65 @@
+.container {
+  color: #271973;
+  font-family: 'Courier New', Courier, monospace;
+}
+
+h1 {
+  font-weight: bolder;
+}
+
 .form-select {
   width: auto;
+}
+
+.movesCounter {
+  display: inline-block;
+  font-weight: bold;
+  animation: bounce .1s ease-in-out;
+}
+
+.emoji {
+  display: inline-block;
+  animation: bounce .1s ease-in-out;
+}
+
+@keyframes bounce {
+  30% {
+    transform: translateY(-4px);
+  }
+  95% {
+    transform: translateY(4px);
+  }
+}
+
+.options {
+  animation: fadeIn .5s cubic-bezier(0.42, 0, 0.44, 1.55) backwards;
```

Comment:
I really like pulling things like bezier curves out into named css variables. It makes the resulting code easier to read (rather than looking at bezier curve values I can just read some descriptive name and have a better understanding of the behavior).

e.g. https://gist.github.com/bendc/ac03faac0bf2aee25b49e5fd260a727d